testing/checkin/runner/ops.rs

```diff
@@ -74,10 +74,12 @@ pub fn op_stats_delete(
 
 pub struct TestObjectWrap {}
 
-impl GarbageCollected for TestObjectWrap {
+unsafe impl GarbageCollected for TestObjectWrap {
```

Comment:
I don't like that we'll have to do this everywhere in Deno 😓

Reply:
oh that will break a lot of stuff. i think we might have to revert this change.

Reply:
I'm in favor of reverting this change